ASAP/ASAP-paper.Rmd

```diff
@@ -32,26 +42,40 @@ lapply(pkgs, library, character.only = T)
 
 # Introduction
 
-The Environmental Stewardship Scheme (ESS) is the current working implementation of agri-environmental scheme (AES) in England. AES are a mechanism that integrates environmental concerns into the European Commission’s Common Agricultural Policy. ESS provides government-financed payments to farmers and land managers in return for a commitment to farming their land with more care for the environment (Smith et al., 2013). Introduced during 2005 and 2006, ESS is a multi-objective scheme with 5 primary objectives: Conservation of wildlife and their habitats; the maintenance and enhancement of landscape quality and character; the protection of the historic environment; the protection of soils and reduction of water pollution; and provision of opportunities for people to visit and learn about the countryside (Natural England, 2011). ESS was established as a ‘broad and shallow’ approach to AES in order to extend its reach to high proportions of the countryside (Amy et al., 2013). As such it is non-competitive and, at some level, open to all farmers and land managers whose land is part of the farmed environment, and is registered in the Rural Land Register (Natural England, 2013b). ESS represents the country’s most widespread approach to environmental management with agreements in place on over 70% of land in the country (Defra, 2013). 
+The Environmental Stewardship Scheme (ESS) is the current working implementation of agri-environmental scheme (AES) in England.
```

Comment:
I think the intro is really good. I've just brushed-up on the wording, and created line breaks after every full stop for ease of tracking changes on GitHub!